packages/gestalt/src/Module.js

```diff
@@ -39,6 +40,10 @@ type Props = {
    * Unique id to identify this Module
    */
   id: string,
+  /**
+   * Sets the size of the Module. See the [size variant](https://gestalt.pinterest.systems.com/web/module#size) to learn more.
+   */
+  size?: 'sm' | 'md' | 'lg',
```

Comment:
Curious. Is this density or size?

Could we just have 2 variants? default | dense

What's the md for? Whats the use case?

In the example we only cover sm for compact density

I think here we could move to density prop. With sizes we would have the same issue with Buttons, how much the sm gets reduced. Or maybe the Density provider overrides all sizes to sm instead of moving values down.

If there's no use case for md, I'd advocate to have default  | compact density variants to avoid inconsistencies.

<img width="674" alt="Screenshot by Dropbox Capture" src="https://github.com/pinterest/gestalt/assets/10593890/54f52354-7d61-4a26-a71f-4207cfd39e9c">

Reply:
I think we had this convo here:
https://pinterest.slack.com/archives/G01GN7CE99S/p1699984230644209?thread_ts=1699974863.546079&cid=G01GN7CE99S